src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js

```diff
@@ -213,6 +217,7 @@ class ActivityFeed extends React.Component<Props, State> {
         } = this.props;
         const { isInputOpen } = this.state;
         const hasCommentPermission = getProp(file, 'permissions.can_comment', false);
+        const hasDeletePermission = getProp(file, 'permissions.can_delete', false);
```

Comment:
Hmm, is this correct? I think annotations will have their own permissions object.